spec/controllers/api/v1/organisations_user_controller_spec.rb

```diff
@@ -1,163 +1,727 @@
+              }.not_to change(OrganisationsUser, :count)
+    
+              expect(response.status).to eq(422)
+              expect(parsed_body['error']).to eq("Missing parameter #{param}")
+            end
+          end
+        end
       end
     end
+  end
+
+  # --------------------
+  # Normal User
+  # --------------------
+
+  context "as a normal user without any permission" do
+    before { generate_and_set_token(user) }
 
-    context 'when a user is already present in the organisation' do
-      it 'does not create organisation_user for duplicate email' do
-        organisations_user = create :organisations_user
-        new_organisations_user_params[:organisation_id] = organisations_user.organisation_id.to_s
-        new_organisations_user_params[:user_attributes][:email] = organisations_user.user.email
+    describe "POST organisations_user" do
+      it "creates new organisations user for himself/herself", :show_in_doc do
         expect {
-          post :create, format: :json, organisations_user: new_organisations_user_params
-        }.to change(OrganisationsUser, :count).by(0)
-         .and change(User, :count).by(0)
-        expect(response.status).to eq(422)
-        expect(subject['errors']).to eq([{'message' => I18n.t("organisations_user_builder.existing_user.present"), 'status' => 422}])
+          post :create, format: :json, organisations_user: organisations_user_payload
+        }.to change(OrganisationsUser, :count).by(1)
+
+        organisations_user = OrganisationsUser.last
+        expect(organisations_user.status).to eq('pending')
+        expect(organisations_user.user_id).to eq(user.id)
+        expect(organisations_user.organisation_id).to eq(organisation.id)
+        expect(organisations_user.position).to eq('Conductor')
+        expect(response.status).to eq(201)
       end
 
-      it 'does not create organisation_user for case insensitive duplicate email' do
-        organisations_user = create :organisations_user
-        new_organisations_user_params[:organisation_id] = organisations_user.organisation_id.to_s
-        new_organisations_user_params[:user_attributes][:email] = organisations_user.user.email.upcase
+      it "defaults to the 'pending' status", :show_in_doc do
         expect {
-          post :create, format: :json, organisations_user: new_organisations_user_params
-        }.to change(OrganisationsUser, :count).by(0)
-         .and change(User, :count).by(0)
-        expect(response.status).to eq(422)
-        expect(subject['errors']).to eq([{'message' => I18n.t("organisations_user_builder.existing_user.present"), 'status' => 422}])
+          post :create, format: :json, organisations_user: organisations_user_payload.except(:status)
+        }.to change(OrganisationsUser, :count).by(1)
+
+        expect(OrganisationsUser.last.status).to eq('pending')
+        expect(response.status).to eq(201)
```

Comment:
There are already specs to prevent a user from modifying his/her status here

```ruby
        [:approved, :expired, :denied].each do |status|
          it "prevents a user from setting his/her status as '#{status}'" do
            expect {
              put :update, id: organisations_user.id, organisations_user: organisations_user_payload.merge(status: status.to_s)
            }.not_to change(OrganisationsUser, :count)

            expect(response.status).to eq(403)
            expect(parsed_body['error']).to eq('Access Denied')
          end
        end
```